db/schema.rb

```diff
@@ -131,6 +130,7 @@
     t.string "postcode"
     t.string "uprn"
     t.json "boundary_geojson"
+    t.text "constraints", default: [], null: false, array: true
```

Comment:
this null false: we always force constraints to exist? Won't this break existing applications if we are not adding defaults? 🤔